### DISCUSSION TITLE
[Feature] Shared album activity notifications

### DISCUSSION THREAD

**Discussion by benjamonnguyen** [immich-app/immich#8938] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

As an enhancement of the multi-user experience, comments, likes, and new pictures added to shared albums should notify participating users. The notifications can be integrated into the account-info-panel.

The multi-user feature sets Immich apart from alternatives like PhotoPrism, and personally the joy of sharing photos with friends and family is a big part of why I self-host Immich!

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by lieberpi** [comment]
I would also love to have notifications of activity, such as comments and likes, in shared albums.
I am testing immich to use to share pictures with family members and it seems weird that one would have to go into the shared album(s) to check and see whether someone has commented on or liked a photo.

It would be awesome if there was a notifications icon by the account profile picture in the top right and if the mobile app would be able to show notifications as well.

**Comment by smartVanture** [comment]
Also voted for this! I share albums with my mom and she comments on it but I never see it, because I won't get notified. 
We do have the bell icon now for sharing activities but I can't find any notification about comments or likes in it. Should be a pretty simple addition, I guess.

**Comment by kingp0dd** [comment]
Yup knowing that's a new item in a started album is nice

**Comment by Oshyan** [comment]
Now that everyone has fled Facebook this is quite critical for my use case. Instagram is where everyone went to share things like family pics, etc., which is depressing as heck because it's not well designed for that use case at all. A simple, effective alternative like Immich is needed, but it has to notify people of interactions for it to give people the full benefits of the connection they want to enjoy around photos. Hopefully this can be prioritized soon as part of the "Enhanced Sharing" work that has recently begun!

**Comment by jo-bzh** [comment]
I’d really love a way to “subscribe” to shared album activity with fine‑grained control over notifications.
My ideal workflow would be:

- Being able to opt in per album to receive notifications
- Choosing the channel: email and/or in‑app/mobile notifications
- Choosing which events to be notified about:
  - new assets added to the album
  - new comments on assets in the album
  - new likes/reactions on assets in the album

This would be especially useful for family albums, where multiple people contribute photos and interact with each other’s content.

I’d be happy to contribute a first implementation for part of this feature, if you think this aligns with the project’s roadmap and design.
